### DISCUSSION TITLE
[Bug]: how to set the date with current weekday

### DISCUSSION THREAD

**Discussion by nousernameavailableanymore** [RafaelGB/obsidian-db-folder#761] [General]
### What happened?

I have a date field as one my columns.
I would like to have the date field (in YAML) popluated with the current date in format yyyy-MM-dd (EEE) using the current computers locale.

So for today that would be:
2023-01-15 (So)

I have configured the setting in the database settings:
![image](https://user-images.githubusercontent.com/45376324/212525669-ce28b042-887e-41ea-9f63-549ae9a6ca4f.png)

But the format set in YAML is shown like this:
![image](https://user-images.githubusercontent.com/45376324/212525689-205ad2e0-3476-4e3b-8f7f-45ddd96e6455.png)

### What platform were you using?

Desktop

### Version of the plugin

3.2.3

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hello @nousernameavailableanymore ,
this behaviour is a decision in favor of an standard with the dates and performance.

All the dates will be saved with ISO format.

There are formats where value is lost and we could not reconstruct a date from them, but if they are saved with this standard it covers any type
